### PR TITLE
Fix compilation and memory leak errors in readme.md

### PR DESCRIPTION
**Pull request by MortyMerr**
I understand that this code is provided just to "show the concept".
But i don't think it's rational argument to give user incorrect code with memory leak. 
Imagine someone copy-pasted it and get leaked viewController.

I didn't add `guard let 'self' = self else { return }` to keep code short.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ RxKeyboard.instance.frame
 
     ```swift
     RxKeyboard.instance.visibleHeight
-      .drive(onNext: { keyboardVisibleHeight in
-        scrollView.contentInset.bottom = keyboardVisibleHeight
+      .drive(onNext: { [weak self] keyboardVisibleHeight in
```
Comment (freak4pc) [inline README.md]:
```suggestion
      .drive(onNext: { [weak scrollView] keyboardVisibleHeight in
```

Comment (twittemb) [reply README.md]: Is it ok to not weakify the scrollView here ?
```swift
.drive(onNext: { [scrollView] keyboardVisibleHeight in
```

**Comment by MortyMerr** [comment]
@twittemb thank u for review. Please check new changes.

**Comment by MortyMerr** [comment]
@freak4pc , @twittemb please gents, let's approve that, i think it's kind of important PR.

**Comment by freak4pc** [comment]
Hey @MortyMerr. Please be patient, we're volunteers, not employees. It's only been 3 days :)

**Comment by MortyMerr** [comment]
Hey @freak4pc . Sorry, didn't want to be rude. Thank u for ur work)
Agree with u, but actually documentation page is the face of the repo. I would disrespect any project for buggy code in docs.

**Pull request merged**